samples/tab-navbar-menu/ts/assets/sample.json

```diff
@@ -0,0 +1,68 @@
+[
+  {
+    "name": "officedev-microsoft-teams-samples-tab-navbar-menu-ts",
+    "source": "officeDev",
+    "title": "Starter solution: Personal tab with Navbar Menu in TypeScript",
```

Comment:
Teams Personal Tab Navbar-Menu Sample TS

Reply:
I have updated the changes.